README.md

```diff
@@ -1,67 +1,69 @@
 
-FAC was founded on the principal of reciprocity. That is, the company does not charge fees for its study programmes, but expects its members to contribute according to their abilities to the viability and development of the company.
+FAC was founded on the principal of reciprocity. That is, the company does not charge fees for its study programmes, but expects its students to contribute to the viability and development of the company in return.
 
-In particular, members are expected to participate in our study programmes and to support our goal of being financially sustainable, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future.
+In particular, members of each cohort of students are expected to take the lead on facilitating the course for the next cohort, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future. Our goal is to be financialy sustainable, and the company cannot survive without the continued support of our graduates.
 
-All members must abide by the following rules of membership.
+In addition to providing a free education, FAC has been successful in finding permanent or freelance work for all of our graduates. Our primary source of income is recruitment fees from companies who hire our graduates, and so graduates are also required to give FAC the opportunity to approach employers on their behalf if we do not already have a relationship with them.
 
-+ To support new members in their studies (and primarily to help with the full-time study programme); 
-+ To participate in and support the study programmes that we provide to the general public;
-+ When considering an application for a position (whether permanent or contract) at a company with which FAC does not have an agency relationship, first give FAC the opportunity to establish contact with the employer for the purposes of initiating an agency relationship;
-+ When considering encouraging another member to apply for a position (whether permanent or contract) at a company with which FAC does not have an agency relationship, first give FAC the opportunity to establish contact with the employer for the purposes of initiating an agency relationship;
-+ To make all other members, as well as members of the public, feel welcome in our space.
+All students must abide by the following rules in exchange for joining FAC:
 
-None of these rules will apply retrospectively.
++ To support new students in their studies (and primarily to help with the full-time study programme); 
++ To participate in and support the study programmes that we provide to the general public;
++ To always involve FAC immediately if they are considering applying for a position (or encouraging another student to apply)
```

Comment:
I mean, it doesn't go so far as to say "We know where you live" (which we do)... 

Can you think of wording that makes it clear how important this rule is without crossing that line? I added "immediately" for the sense of urgency, and "always" for the sense of importance - maybe it could live without "always" as that does carry the implication that this is an ongoing requirement when technically it's probably only relevant to their first job